docs/source/api/entities.md

```diff
@@ -0,0 +1,329 @@
+3. Captures Sync issues and reports those with a Status E-Mail to a central E-Mail Address (Sync error reporting)
+4. Performance: Only downloads assets that are not downloaded yet and tries to optimize the actual write calls to the database by using MongoDB bulk operations, making sync processing fast
+5. Push-based: Immediate representation of changes
+
+In some situations, it's propably wise to develop a sync microservice: You have a source system that
+- generates "Pull-based" data feeds, or
+- can not adopt to the JSON described below or
+- needs to merge data from different systems
+
+### Endpoint
+
+The main entry point to send import events is:
+Mutation.sendBulkImportEvents. The mutation takes an array of events that consist out of a type, an operation and a payload.
+
+Supported entity types:
+- PRODUCT
+- ASSORTMENT
+- FILTER
+
+Supported operations:
+- CREATE
+- UPDATE
+- REMOVE
+
+All Events follow this JSON Structure:
+
+```json
+{
+  "entity": "ENTITY TYPE",
+  "operation": "OPERATION TYPE",
+  "payload": { ... }
+}
+```
+
+Always try to send as many events at a time, so Unchained can optimize write operations.
+
+## JSON Reference
+
+### Entity Type: Product
+
+Set by unchained:
+- authorId,
+- slug history
+- _id, contentHash, created & updated if not provided
+
+```json
+{
+  "entity": "PRODUCT",
+  "operation": "CREATE",
+  "payload": {
+    "specification": {
+      "_id": "A",
+      "created": null,
+      "updated": null,
+      "sequence": "",
+      "tags": ["nice"],
+      "type": "SimpleProduct",
+      "status": "ACTIVE",
+      "published": "2020-01-01T00:00Z",
+      "commerce": {
+        "salesUnit": "ST",
+        "salesQuantityPerUnit": "1",
+        "pricing": [
+          {
+            "isTaxable": true,
+            "isNetPrice": true,
+            "countryCode": "CH",
+            "currencyCode": "CHF",
+            "amount": 10000,
+            "maxQuantity": null
+          }
+        ]
+      },
+      "warehousing": {
+        "baseUnit": "ST",
+        "sku": ""
+      },
+      "supply": {
+        "weightInGram": 0,
+        "heightInMillimeters": 0,
+        "lengthInMillimeters": 0,
+        "widthInMillimeters": 0
+      },
+      "proxy": {
+        "assignments": [
+          {
+            "vector": {
+              "color": "red"
+            },
+            "productId": "B"
+          }
+        ]
+      },
+      "plan": {
+        "billingInterval": "daily",
+        "billingIntervalCount": 1,
+        "usageCalculationType": "metered",
+        "trialInterval": "daily",
+        "trialIntervalCount": 1
+      },
+      "bundleItems": [
+        {
+          "productId": "c",
+          "quantity": 1,
+          "configuration": [
+            {
+              "key": "greeting",
+              "value": "For my Darling"
+            }
+          ]
+        }
+      ],
+      "meta": {},
+      "content": {
+        "de": {
```

Comment:
It would probably be better to use the longer lanuage code to be future proof? Like `de_CH`? Or accept both explicitly.

Reply:
There is no rules on how languages can be added so both are accepted, you could add a Language with an isoCode of "klingon" and use that here.